### PR TITLE
feat(seeding): add sample seeding for prod and dev

### PR DESCRIPTION
**Pull request by MatoPlus**
### Notion ticket link

[Seeding Database](https://www.notion.so/uwblueprintexecs/Seeding-Database-32402df8657d407285565f2976f4c009)

### Implementation description

- Add sample seeding for classes, programs, and program translations which refects the records in staging.

### Steps to test

1. Create a new DB URL
2. run `yarn`
2. run the sql script against the DB (using the heroku client)
3. do prisma introspect and generate
4. run `npx ts-node -O {\"module\":\"CommonJS\"} prisma/seed.ts` 
5. run `yarn dev` and ensure pages with program are populated

### What should reviewers focus on?

- content of sample data
- how seeding is done
- style

### Checklist

-   [X] My PR name is descriptive and in imperative tense
-   [X] I have run the linter
-   [X] I have requested a review from the PL, and/or other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: prisma/dev-seeds/class.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+                id,
+            },
+            update: rest,
+            create: {
+                id,
+                updatedAt,
+                ...rest,
+            },
+        });
+        console.log({ classUpsert });
```
Comment (mahadzaryab1) [inline prisma/dev-seeds/class.ts]: nit: can remove if not needed anymore

File: prisma/dev-seeds/program-translation.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            update: rest,
+            create: {
+                programId,
+                language,
+                createdAt,
+                updatedAt,
+                ...rest,
+            },
+        });
+        console.log({ translationUpsert });
```
Comment (mahadzaryab1) [inline prisma/dev-seeds/program-translation.ts]: same comment as above

File: prisma/dev-seeds/program.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            "https://images.squarespace-cdn.com/content/v1/5e83092341f99d6d384777ef/1608341017251-K0Q0U7BC37SQ5BGCV9G0/IMG_6646.jpg?format=750w",
+        startDate: new Date(),
+        endDate: new Date(new Date().setDate(new Date().getDate() + 7)),
+        isArchived: false,
+        createdAt: new Date(),
+        updatedAt: null,
+    },
+    {
+        id: 2,
+        price: 0,
```
Comment (mahadzaryab1) [inline prisma/dev-seeds/program.ts]: it might be worth inserting a non-zero price here for testing purposes

File: prisma/dev-seeds/program.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+                id,
+            },
+            update: rest,
+            create: {
+                id,
+                updatedAt,
+                ...rest,
+            },
+        });
+        console.log({ programUpsert });
```
Comment (mahadzaryab1) [inline prisma/dev-seeds/program.ts]: same comment as above

File: prisma/prod-seeds/program-translation.ts [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+            update: rest,
+            create: {
+                programId,
+                language,
+                createdAt,
+                updatedAt,
+                ...rest,
+            },
+        });
+        console.log({ translationUpsert });
```
Comment (mahadzaryab1) [inline prisma/prod-seeds/program-translation.ts]: same comment as above

File: prisma/prod-seeds/program.ts [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+                id,
+            },
+            update: rest,
+            create: {
+                id,
+                updatedAt,
+                ...rest,
+            },
+        });
+        console.log({ programUpsert });
```
Comment (mahadzaryab1) [inline prisma/prod-seeds/program.ts]: same comment as above

**Review by mahadzaryab1 — APPROVED** [review]
Added a few minor nits, otherwise LGTM!

File: prisma/dev-seeds/index.ts [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import classUpsert from "./class";
+import programUpsert from "./program";
+import programTranslationsUpsert from "./program-translation";
+
+/**
+ * Seed the Development environment
+ */
+export default async function seedDev(): Promise<void> {
```
Comment (feng-eric) [inline prisma/dev-seeds/index.ts:12]: I think there might be a need to seed a teacher user as well. Can you check the current staging database to see if we have a teacher user. If there is one, I believe we need it as I think we use the name field of the teacher. @MatoPlus

Comment (MatoPlus) [reply prisma/dev-seeds/index.ts:12]: You're absolutely right 👍 . I was working the page to do parent class registration and the model needs some more sample data in `Teacher` along with `ClassTranslation`. I'll try to add those data along with anything else that may seem important later today.

**Review by gregmaxin — APPROVED** [review]
Testing and works well

Looks good - although I am new to this project and not sure if anything is missing

**Review by feng-eric — APPROVED** [review]
LGTM!

**Pull request merged**